schema-nft-marketplace.graphql

```diff
@@ -0,0 +1,261 @@
+# Subgraph Schema: NFT Marketplace
+# Version: 0.0.1
```

Comment:
From past experience, I think probably a good idea to start with some thing less than `1.0.0`, and only change to `1.0.0` after we are done with the first subgraph, otherwise there might be more changes along the way and it'd be difficult to track versions